AmplifyPlugins/Auth/AWSAuthPlugin/Dependency/AuthenticationProviderAdapter.swift

```diff
@@ -16,13 +16,14 @@ class AuthenticationProviderAdapter: AuthenticationProviderBehavior {
         self.awsMobileClient = awsMobileClient
     }
 
-    func signInUsername() -> Result<String, AuthError> {
+    func currentUser() -> AuthUser? {
```

Comment:
rename to `getCurrentUser`, or make this a property.

Reply:
Updated